CONVENTIONS.md

```diff
@@ -0,0 +1,75 @@
+MCP Naming Conventions
+======================
+
+In order to guarantee a consistent style and theme across MCP names, the following conventions
+should be adhered to when mapping new things or proposing replacements for existing mappings.
+Please refer to this document whenever you do not know how to format or structure a name.
+
+Getting Started
+---------------
+ - **If you do not know what to name something then *do not name it*. Let someone else do so.**
+ - Names should reflect intent, *not* implementation. Check usages rather than just looking at the definition.
```

Comment:
It would also be good to mention when naming methods to not only check usages, but also still consider implementation, especially when multiple types are providing implementations. This helps to further reveal intended semantics.